### DISCUSSION TITLE
copilot.vim - allow overriding copilot end point

### DISCUSSION THREAD

**Discussion by tmc** [community/community#74468] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

I'd love to be able to take an approach such as https://github.com/danielgross/localpilot with copilot.vim, @tpope any thoughts on making this overridable?

**Comment by tpope** [comment, marked as answer]
This isn't officially supported, but the environment variable `AGENT_DEBUG_OVERRIDE_PROXY_URL` does the same thing as the VS Code option `debug.DebugOverrideProxyUrl`.
